library/perl

```diff
@@ -1,21 +1,20 @@
-# maintainer: Peter Martini <PeterCMartini@GMail.com> (@PeterMartini)
+# this file is generated via https://github.com/docker-library/python/blob/c5b36c38823ca6a1301ad39cbf3ece75d77c5600/generate-stackbrew-library.sh
```

Comment:
Python eh? :wink:

Also, if all the sections use the same commit, you can put `GitCommit` in the global section (the first one) and it will apply to all.